src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -290,7 +265,7 @@ protected override void OnScriptFileChanged(object sender, FileSystemEventArgs e
                     try
                     {
                         fileTraceWriter.Info(text);
-                        traceWriter.Info(text);
+                        invocationContext.TraceWriter.Info(text);
```

Comment:
At one point, yes. But I backed that out with one of my later commits -- it wasn't necessary anymore as the core changes do everything we need.

---
In reply to: [77705826](https://github.com/Azure/azure-webjobs-sdk-script/pull/604#discussion_r77705826) [](ancestors = 77705826)